src/java.base/share/classes/java/net/InterfaceAddress.java

```diff
@@ -45,7 +45,7 @@ public class InterfaceAddress {
      * Package private constructor. Can't be built directly, instances are
```

Comment:
The comment is outdated.  This constructor is actually called via JNI like `(*env)->NewObject(env, ni_ibcls, ni_ibctrID)` in `NetworkInterface.c`.  I think we need to consult a network engineer to decide what is the best way to comment on this constructor.